xapian-applications/omega/templates/query

```diff
@@ -108,9 +108,9 @@ $if{$field{size},<span title="$html{$field{size}} bytes">Size: <b>$html{$filesiz
 </div>
 </div>
 </td>
-<td><b><a href="$if{$opt{clicklog},$opt{clicklogurl},$html{$field{url}}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
+<td><b><a href="$html{$opt{clickurl}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
 <small>$snippet{$field{sample}}</small><br>
-<a href="$if{$opt{clicklog},$opt{clicklogurl},$html{$field{url}}}">$html{$field{url}}</a><br>
+<a href="$html{$opt{clickurl}}">$prettyurl{$html{$field{url}}}</a><br>
```

Comment:
I think html escaping isn't required in `$prettyurl{$html{$field{url}}}` instead we probably just want `$prettyurl{$field{url}}` here?